### PR TITLE
Index doesn't know about ReadService

### PR DESCRIPTION
**Pull request by miklos-da**
Summary of changes:
* `JdbcIndex` doesn't depend on `ReadService`.

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/sandbox/src/main/scala/com/digitalasset/platform/index/JdbcIndex.scala [hunk 1]
```diff
@@ -26,11 +26,8 @@ object JdbcIndex {
     ReadOnlySqlLedger(jdbcUrl, Some(ledgerId), metrics).map { ledger =>
       new LedgerBackedIndexService(MeteredReadOnlyLedger(ledger, metrics), participantId) {
         override def getLedgerConfiguration(): Source[v2.LedgerConfiguration, NotUsed] =
-          // FIXME(JM): This is broken. We should not use ReadService in Ledger API Server,
-          // The indexer should on start set the default configuration.
-          readService.getLedgerInitialConditions().map { cond =>
-            v2.LedgerConfiguration(cond.config.timeModel.minTtl, cond.config.timeModel.maxTtl)
-          }
+          // FIXME(JM): The indexer should on start set the default configuration.
```
Comment (miklos-da) [inline ledger/sandbox/src/main/scala/com/digitalasset/platform/index/JdbcIndex.scala]: @dajmaki Do you think we can remove this FIXME?

Comment (dajmaki) [reply ledger/sandbox/src/main/scala/com/digitalasset/platform/index/JdbcIndex.scala]: As long as we track this properly and fix it and the other FIXME that got removed above.

**Comment by gerolf-da** [comment]
Changed title and description from `Indexer` to `Index`.

File: ledger/sandbox/src/main/scala/com/digitalasset/platform/index/JdbcIndex.scala (hunk 1 above)
Comment (miklos-da) [reply ledger/sandbox/src/main/scala/com/digitalasset/platform/index/JdbcIndex.scala]: OK, will keep it.

**Pull request merged**